index.html

```diff
@@ -57,5 +58,204 @@ <h3 class="contact-title">let's connect</h3>
 				</div>
 			</div>
 		</header>
+
+		<!-- Work section -->
+		<section class="work">
+			<div class="single-card">
+				<img
+					src="/images/SnapshootPortfolio-1.png"
+					alt="Portfolio 1 image"
+					class="p-img"
+				/>
+				<div class="about-portfolio">
+					<h1>Tonic</h1>
+					<div class="info-portfolio">
+						<ul class="list">
+							<li class="li">canopy</li>
+							<li>Back End Dev</li>
+							<li>2015</li>
+							<li class="li"></li>
+							<li class="li"></li>
+						</ul>
+					</div>
+					<p>
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required.
+					</p>
+
+					<ul class="tech-used">
+						<li class="tech">html</li>
+						<li class="tech">css</li>
+						<li class="tech">javaScript</li>
+					</ul>
+				</div>
+				<a href="#" class="btn">See Projects</a>
+			</div>
+
+			<div class="single-card">
+				<img
+					src="/images/SnapshootPortfolio-2.png"
+					alt="Portfolio 1 image"
+					class="p-img"
+				/>
+				<div class="about-portfolio">
+					<h1>Multi-Post Stories</h1>
```

Comment:
- [ ] Kindly change the `h1` tag to any other heading tag between `h2` and `h6`.

Reply:
Done